WordPress/Jetpack/Classes/ViewRelated/WordPress Migration/Welcome/MigrationWelcomeBlogTableViewCell.swift

```diff
@@ -0,0 +1,103 @@
+import UIKit
+import SwiftUI
+
+final class MigrationWelcomeBlogTableViewCell: UITableViewCell, Reusable {
```

Comment:
Again, totally not opposed to this, but wondering if we could just reuse the existing `UITableViewCell` that we currently have in the site picker, and looks like this one?

Reply:
I tried using `WPBlogTableViewCell` ( the one in the site picker screen ) but it doesn't look exactly like the design. The image size, spacing between elements were different. We could make `WPBlogTableViewCell` more customizable so it matches the Welcome Screen design, but I think that should be done in another PR.

Reply:
As discussed on Slack, we can keep the new one, because customizing the existing one takes more code than this and will not create a general component anyway.